packages/docs/src/examples/v-stepper/misc-horizontal.vue

```diff
@@ -1,21 +1,120 @@
 <template>
-  <v-stepper value="1">
-    <v-stepper-header>
-      <v-stepper-step step="1">
-        Select campaign settings
-      </v-stepper-step>
+  <v-stepper
+    v-model="step"
+    :items="items"
+    show-actions
```

Comment:
I think there needs to be some easy way of

1. Making actions not circular (removing previous on first, next on last)
2. Adding some custom button/content in place of previous on first, next on last

In this example there needs to be a "place order" button on last screen, and possibly a "back" or "cancel" button on first one.